src/cls/rgw/cls_rgw_client.h

```diff
@@ -463,13 +463,13 @@ void cls_rgw_encode_suggestion(char op, rgw_bucket_dir_entry& dirent, bufferlist
 void cls_rgw_suggest_changes(librados::ObjectWriteOperation& o, bufferlist& updates);
 
 /* usage logging */
-int cls_rgw_usage_log_read(librados::IoCtx& io_ctx, string& oid, string& user,
+int cls_rgw_usage_log_read(librados::IoCtx& io_ctx, string& oid, string& user, string& subuser,
```

Comment:
The `const` is fine for me. But the `std` namespace seems not consistent with code around? 

If we introduce the `std`  for `subuser` , what about `user` , `oid` even the similar arguments for `cls_rgw_usage_log_trim`?  Do you think the cleanup is necessary?